src/benchmark_environments/mujoco.py

```diff
@@ -0,0 +1,76 @@
+"""Adaptation of MuJoCo environments for IRL."""
+
+import gym
+import gym.envs
+from gym.envs.mujoco import (
+    ant_v3,
+    half_cheetah_v3,
+    hopper_v3,
+    humanoid_v3,
+    swimmer_v3,
+    walker2d_v3,
+)
+
+
+def _include_position_in_observation(cls):
+    old_init = cls.__init__
+
+    def new_init(*args, **kwargs):
+        kwargs["exclude_current_positions_from_observation"] = False
+        old_init(*args, **kwargs)
+
+    cls.__init__ = new_init
+    return cls
+
+
+def _no_early_termination(cls):
+    cls.done = False
```

Comment:
I don't understand how this works. `done` is computed and returned by `step`, it's not a class attribute.

I think you'll need to override `step` similar to the above decorator.

Reply:
So, for [hopper_v3](https://github.com/openai/gym/blob/master/gym/envs/mujoco/hopper_v3.py#L110), [humanoid_v3](https://github.com/openai/gym/blob/master/gym/envs/mujoco/humanoid_v3.py#L125), [walker2d_v3](https://github.com/openai/gym/blob/master/gym/envs/mujoco/walker2d_v3.py#L102) and [ant_v3](https://github.com/openai/gym/blob/master/gym/envs/mujoco/ant_v3.py#L96), `done` is calculated by `done = self.done`, (in which self.done is class method defined with a property decorator), and thus `done` should now be always be `False` with the change; and for [half_cheetah_v3](https://github.com/openai/gym/blob/master/gym/envs/mujoco/half_cheetah_v3.py#L48) and [swimmer_v3](https://github.com/openai/gym/blob/master/gym/envs/mujoco/swimmer_v3.py#L46), done is calculated as `done = False`, so adding `cls.done = False` should be a no-op for these two.

I guess this is does not immediately generalize to the other environments, but I felt it was simple as a first step and that it could be changed if desired after environments were adapted.

But from my understanding, it should be working - I might be misunderstanding something though.